sdk/master_board_sdk/CMakeLists.txt

```diff
@@ -130,3 +141,8 @@ if(BUILD_TESTING)
   add_unit_test(test_protocol tests/test_protocol.cpp)
   target_link_libraries(test_protocol PRIVATE Catch2::Catch2WithMain)
 endif()
+
+# --- Call ROS-2 if possible ---
+if(ROS_VERSION EQUAL 2)
+  ament_package()
```

Comment:
if this overwrites the master-board-sdkConfig.cmake we could have issues with the downstream dependencies as jrl-cmakemodules properly writes the dependecies in the Config.cmake.

But for ament to do so you need to call another ament function, I believ it's `ament_export_dependencies(blabla)`.

Maybe you already checked that.